frontend/src/components/TaskMap.tsx

```diff
@@ -279,19 +280,38 @@ export const TaskMap: FC<{
           dispatch(roadmapsActions.setTaskmapPosition(viewport));
         }}
       >
-        <Controls showInteractive={false} showZoom={false}>
-          <InfoTooltip title={t('Taskmap-tooltip')}>
-            <InfoIcon
-              className={classes(css.info, css.tooltipIcon, css.infoIcon)}
-            />
-          </InfoTooltip>
-          <InfoTooltip title={t('Reset taskgroup positions')}>
-            <RestartAltIcon
-              className={classes(css.restart)}
-              onClick={resetCanvas}
-            />
-          </InfoTooltip>
-        </Controls>
+        <div className={classes(css.controlsContainer)}>
+          <Controls
+            className={classes(css.controls)}
+            showInteractive={false}
+            showZoom={false}
+            showFitView={false}
+          >
```

Comment:
Do we need the Controls component anymore as each control is false? 
Without this, there is no need for relative control styles (example below)

```
.controlsContainer {
  @extend .layout-row;
  justify-content: flex-end;
  align-items: center;
  margin-right: 10px;
  margin-top: 10px;
  gap: 7px;
}

.controlButton {
  cursor: pointer;
}
```

Reply:
I also tried removing this as I noticed that I disabled all the functionalities but the buttons don't seem to work on the canvas unless they're inside the <Controls> .. 😅